templates/common/baremetal/files/NetworkManager-static-dhcp.yaml

```diff
@@ -0,0 +1,30 @@
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/30-static-dhcp"
+contents:
+  inline: |
+    #!/bin/bash
+    set -ex
+
+    if [ ${DHCP4_DHCP_LEASE_TIME:-0} -lt 4294967295 ]
+    then
+        echo "Not an infinite lease. Ignoring."
```

Comment:
Should probably go to stderr

Reply:
Ah, yeah I'll update this to match the logging in the other dispatcher scripts.